### PR TITLE
LRAC-10692 Get the title to be sent in the download event

### PR DESCRIPTION
**Pull request by interaminense**
JIRA Issue: [LRAC-10692](https://issues.liferay.com/browse/LRAC-10692)

### CODE REVIEW

**Comment by interaminense** [comment]
cc: @kienD

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-analytics/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp [hunk 1]
```diff
@@ -55,15 +60,24 @@
...
+				title: title || decodeURIComponent(match[3].replace(/\+/gi, ' ')),
 				version: getParameterValue('version'),
 			});
 		}
 	}
 
 	function handleDownloadClick(event) {
 		if (window.Analytics) {
-			if (
+			if (event.target.nodeName.toLowerCase() === 'a') {
```
Comment (kienD) [inline modules/apps/document-library/document-library-analytics/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp:71]: Why did we move these two if statements up?

**Review by kienD — CHANGES_REQUESTED** [review]
just a question

**Comment by interaminense** [comment]
ci:foward

**Comment by interaminense** [comment]
ci:forward

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**